src/test/article.spec.js

```diff
@@ -0,0 +1,267 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import 'chai/register-should';
+import 'chai/register-expect';
+import BaseRepository from '../repository/base.repository';
+import db from '../database/models';
+import app from '../index';
+import {
+  articleSample,
+  validArticleDetails,
+  articleWithNoContent,
+  articleWithNoTitle,
+  articleWithNoImage,
+  articleWithNoDescription
+} from '../database/mockdata/articledata';
+import utils from '../helpers/utils';
+
+// configure chai to use expect
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const articleEndpoint = '/api/v1/articles';
+
+describe('POST/GET/PUT/DELETE CRUD ACTIVITIES ON ARTICLES CONTROLLER', () => {
```

Comment:
Hello @tobechukwuobitube , I do not think you need this extra description since each method already has it's own description

Reply:
Thank you for this review. Please, can you suggest a better way to go about this the `description`?